web/src/vdom.ts

```diff
@@ -41,9 +60,7 @@ export function render_tag(tag) {
     */
     const opts = tag.opts;
     const tag_name = tag.tag_name;
-    const attr_str = opts.attrs
-        .map((attr) => " " + attr[0] + '="' + _.escape(attr[1]) + '"')
-        .join("");
+    const attr_str = [...opts.attrs].map((attr) => ` ${attr[0]}="${_.escape(attr[1])}"`).join("");
```

Comment:
We can probably avoid this by typing `opts.attrs` as `[string, string][]`. Could you verify that the callers are already passing arrays for `Options`?

Reply:
Function `update_attrs` requires `Iterable<[string, string]>` because `Map()` constructure requires it. Thats why I was making `Option.attrs` itererable as well. I just tested it and I can make `Option.attrs` as you suggested while keep type compatible in `update_attrs` function. Resolved!